tools/dockerise/__init__.py

```diff
@@ -74,6 +74,8 @@ def build_platform(platform):
             "BUILDKIT_INLINE_CACHE=1",
             "--build-arg",
             "platform={}".format(platform if platform != "buildkit" else "generic"),
+            "--build-arg",
+            "user_uid={}".format(os.getuid()),
```

Comment:
[`os.getuid()`](https://docs.python.org/3/library/os.html#os.geteuid) is not available on Windows (natively). Though I suppose we can start assuming WSL?

Reply:
If you want to add some code you could just set it up so that on windows which is not wsl, you can just ignore this whole build argument (or set it to the default of 1000)

Reply:
Yeah, plus there are other things in the code that won't work without WSL. We can fix those all in the future if necessary.